openscad.html.markdown

```diff
@@ -0,0 +1,120 @@
+---
+language: openscad
+filename: leanopenscad.scad
+contributors:
+    - ["Thomas Preston", "https://github.com/tompreston/"]
+---
+
+Draw 3D models with code using [OpenSCAD](https://openscad.org/).
+
+```openscad
+// Single-line comments start with //
+
+/*
+Multi-line comments look like this.
```

Comment:
In the other multi-line comments, you have a * at the beginning of the line. Is this optional?

Reply:
[It's optional](https://en.wikibooks.org/wiki/OpenSCAD_User_Manual/General#Comments)

In retrospect, it's a bit confusing having the two kinds of comments at all. I've removed the multi-line comments completely.